packages/functional-tests/pages/singupTokenCode.ts

```diff
@@ -0,0 +1,9 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { BaseTokenCodePage } from './baseTokenCode';
+
+export class SignupTokenCodePage extends BaseTokenCodePage {
```

Comment:
It might be clearer to follow the route name, and call the class and file `ConfirmSignupCodePage`?

Reply:
I can change it. I was on the fence how to name it, since SigninTokenCodePage already existed. I'll rename though. Better for it be correct. There's another similar naming thing I ran into that I'll fix as well.

Reply:
This has been changed as recommended.